hedera-node/src/main/java/com/hedera/services/store/tokens/unique/UniqueStore.java

```diff
@@ -32,7 +32,10 @@
  */
 public interface UniqueStore extends TokenStore {
 
-	ResponseCodeEnum mint(final TokenID tId, String memo, RichInstant creationTime);
+//	ResponseCodeEnum mint(final TokenID tId, String memo, RichInstant creationTime);
+	ResponseCodeEnum mint(final TokenMintTransactionBody txBody, final RichInstant creationTime);
+	ResponseCodeEnum mintProvisional(final TokenMintTransactionBody request, final RichInstant creationTime);
+	ResponseCodeEnum commitProvisional();
```

Comment:
How about to use default interface methods here, and include the  exception throwing for UnsupportedOperationException in it. That way we can remove all the boilerplate code from ExceptionalUniqueTokenStore

Reply:
Will consider it after i finish mint OP